lib/libcompat.h

```diff
@@ -45,6 +45,11 @@
 #define CK_ATTRIBUTE_NORETURN
 #endif /* GCC 2.5 */
 
+/* Actually GCC 4.6.4 */
```

Comment:
Oh, you did move it to the header.

Reply:
I mean to improve the `GCC_VERSION_AT_LEAST(major, minor)` macro to `GCC_VERSION_AT_LEAST(major, minor, patch)` because the patch number is useful in some circumstances, such as here.